docker/build-tools/build-and-push.sh

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+HUB=gcr.io/istio-testing
```

Comment:
as a followon - would be nice to just import the HUB from the environment if it is set.